typescript/web/src/components/dataset-classes/insert-many-classes-modal/insert-many-classes-modal.tsx

```diff
@@ -0,0 +1,158 @@
+import { ApolloError } from "@apollo/client";
```

Comment:
I've not seen any Story, Test or Cypress test for this new component. Can we have them?

Reply:
Story and Test done, but not Cypress: we currently don't do any Cypress tests on the dataset classes list tab, I think we should open a dedicated issue for that